subprojects/build-comparison/src/main/java/org/gradle/api/plugins/buildcomparison/gradle/internal/DefaultGradleBuildInvocationSpec.java

```diff
@@ -34,6 +35,7 @@
     private List<String> tasks = new LinkedList<String>();
     private List<String> arguments = new LinkedList<String>();
 
+    @Inject
```

Comment:
The javadocs for `newInstance()` don't say anything about this, we should mention it.